doc/languages-frameworks/vim.section.md

```diff
@@ -234,16 +234,48 @@ Finally, there are some plugins that are also packaged in nodePackages because t
 
 ### Testing Neovim plugins {#testing-neovim-plugins}
 
-`nvimRequireCheck=MODULE` is a simple test which checks if Neovim can requires the lua module `MODULE` without errors. This is often enough to catch missing dependencies.
+#### neovimRequireCheck {#testing-neovim-plugins-neovim-require-check}
+`neovimRequireCheck` is a simple test which checks if Neovim can requires lua modules without errors. This is often enough to catch missing dependencies.
 
-This can be manually added through plugin definition overrides in the [overrides.nix](https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/editors/vim/plugins/overrides.nix).
+It accepts a single string for a module, or a list of module strings to test.
+- `nvimRequireCheck = MODULE;`
+- `nvimRequireCheck = [ MODULE1 MODULE2 ];`
```

Comment:
My personal preference would have been to accept juste one format to avoid surprising the user "it's a list here but not there ?", plus it makes implementation easier. Like always accept a list and mark it plural:
`nvimRequireChecks = []`